Assets/MixedRealityToolkit-SDK/Features/Input/Scripts/PointerClickHandler.cs

```diff
@@ -0,0 +1,66 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using Microsoft.MixedReality.Toolkit.Internal.Definitions.InputSystem;
+using Microsoft.MixedReality.Toolkit.Internal.EventDatum.Input;
+using Microsoft.MixedReality.Toolkit.Internal.Interfaces.InputSystem.Handlers;
+using UnityEngine;
+
+namespace Microsoft.MixedReality.Toolkit.SDK.Input
+{
+    /// <summary>
+    /// This component handles pointer clicks from all types of input sources.<para/>
+    /// i.e. a primary mouse button click, motion controller selection press, or hand tap.
+    /// </summary>
+    public class PointerClickHandler : BaseInputHandler, IMixedRealityPointerHandler
+    {
+        [SerializeField]
+        [Tooltip("The input actions to be recognized on pointer up.")]
+        private InputActionEventPair onPointerUpActionEvents;
```

Comment:
This is described as action eventS, yet there is only one pair defined... do we *really* want to support registering multiple events on a single action or should this be a collection

(same with the other two IAEP fields)